.openpublishing.redirection.json

```diff
@@ -50936,14 +50936,144 @@
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/batchai.md",
+        "redirect_url": "/java/api/overview/azure/batch-ai?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/cdn.md",
+        "redirect_url": "/java/api/overview/azure/content-delivery-network?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/cosmosdb.md",
+        "redirect_url": "/java/api/overview/azure/cosmos-db?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/datalakeanalytics.md",
+        "redirect_url": "/java/api/overview/azure/data-lake-analytics?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/entitysearch.md",
+        "redirect_url": "/java/api/overview/azure/entity-search?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/eventgrid.md",
+        "redirect_url": "/java/api/overview/azure/event-grid?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/eventhubs.md",
+        "redirect_url": "/java/api/overview/azure/event-hubs?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/keyvault.md",
+        "redirect_url": "/java/api/overview/azure/key-vault?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/rediscache.md",
+        "redirect_url": "/java/api/overview/azure/redis-cache?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/servicebus.md",
+        "redirect_url": "/java/api/overview/azure/service-bus?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/spellcheck.md",
+        "redirect_url": "/java/api/overview/azure/spell-check?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/trafficmanager.md",
+        "redirect_url": "/java/api/overview/azure/traffic-manager?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/legacy/cdn.md",
+        "redirect_url": "/java/api/overview/azure/content-delivery-network?view=azure-java-legacy",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/appservice.md",
+        "redirect_url": "/java/api/overview/azure/app-service?view=azure-java-stable",
```

Comment:
These are duplicated but the moniker is not updated. The `?view=` should point to the new moniker.

Reply:
These are not duplicated. Not sure why I was thinking that. These monikers match correctly to the paths :)